backend/editor/entries.py

```diff
@@ -270,20 +270,30 @@ async def list_projects(self, status=None):
             "MATCH (n:PROJECT)",
             "OPTIONAL MATCH (error_node:ERRORS {branch_name: n.branch_name, id: n.id})",
         ]
+
         params = {}
         if status is not None:
             # List only projects matching status
             query.append("WHERE n.status = $status")
             params["status"] = status
+
         query.extend(
             [
                 "WITH n, size(error_node.errors) AS errors_count",
                 "RETURN n{.*, errors_count: errors_count}",
                 "ORDER BY n.created_at",
             ]
         )
-        result = await get_current_transaction().run("\n".join(query), params)
-        return await async_list(result)
+
+        queryResult = await get_current_transaction().run("\n".join(query), params)
```

Comment:
Remember you are in the Python side, here you have to use snake_case, CamelCase is reserved to classes.

Reply:
done, replaced with snake_case